syntax = "proto3";
package objectlocation;

import "AlgorithmConfig_conf.proto";

// 姿态估计模型详细配置
message TaskConfig {
    float iou_thres = 1;        // IoU阈值
    int32 num_keys = 2;         // 关键点数量
    float bucket_size = 3;      // 桶大小
    float max_distance = 4;     // 最大距离
    bool run_status = 5;        // 代码运行方式
}

// 姿态估计任务配置参数
message ObjectLocationConfig {
    TaskConfig task_config = 1;
    common.ModulesConfig modules_config = 2;
}
